CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - `normalize_performance_score` stores 0 to 1 cdf score instead of weighted score for each performance score component. ([#2734](https://github.com/getsentry/relay/pull/2734))
 - Add Bytespider (Bytedance) to web crawler filter. ([#2747](https://github.com/getsentry/relay/pull/2747))
+- Add Mixed JS/Android Profiles events processing. ([#2706](https://github.com/getsentry/relay/pull/2706))
```

Comment:
Please, merge `master` in and if needed move this record to `Unreleased` section.